release-notes/opensearch-anomaly-detection.release-notes-2.0.0.0-rc1.md

```diff
@@ -0,0 +1,44 @@
+## Version 2.0.0.0-rc1 Release Notes
+
+Compatible with OpenSearch 2.0.0
```

Comment:
should this be 2.0.0-rc1 on this line? @peterzhuamazon

Reply:
I think it should be @aksingh-es

Reply:
+1

Reply:
As discussed offline, this should be changed to 2.0.0-rc1 before merging.

Reply:
2.0.0-rc1